packages/schemas/src/barcodes/helper.ts

```diff
@@ -170,6 +170,27 @@ export const createBarCode = async (arg: {
     bwipjs.toCanvas(canvas, bwipjsArg);
     const dataUrl = canvas.toDataURL('image/png');
     res = b64toUint8Array(dataUrl).buffer as Buffer;
+    // @ts-ignore
+  } else if (typeof WorkerGlobalScope !== 'undefined' && self instanceof WorkerGlobalScope) {
+    // Web Worker environment
+    let canvas
```

Comment:
It will be simpler if done as follows.

```suggestion
    const canvas = new OffscreenCanvas(width, height);
```

Reply:
This creates an error: TS2588: Cannot assign to canvas because it is a constant.